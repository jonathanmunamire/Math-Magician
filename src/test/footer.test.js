import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/footer';

describe('test if the page Footer renders components', () => {
  test('test if the page Footer is rendered', () => {
    const footer = renderer
      .create(<Footer />)
      .toJSON();

    expect(footer).toMatchSnapshot();
  });
});
