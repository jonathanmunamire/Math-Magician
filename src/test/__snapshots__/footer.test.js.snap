// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test if the page Footer renders components test if the page Footer is rendered 1`] = `
<footer
  className="footer"
>
  <p
    className="footer_text"
  >
    Copyright © 2022 | Designed by 
    <span>
      Jonathan MUNAMIRE
    </span>
  </p>
</footer>
`;
